microbench/pom.xml

```diff
@@ -43,17 +42,6 @@
           <family>linux</family>
         </os>
       </activation>
-      <properties>
-        <epoll.arch>${os.detected.arch}</epoll.arch>
-      </properties>
-      <dependencies>
-        <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>netty-transport-native-epoll</artifactId>
-          <version>${project.version}</version>
-          <classifier>${jni.classifier}</classifier>
```

Comment:
So this is not needed anymore?

Reply:
A nevermind I see you removed the benchmark that needed epoll.

Reply:
There are no benchmarks which use the epoll transport any more. The only benchmark which used the epoll transport was removed in this PR. As discussed in https://github.com/netty/netty/pull/7417#discussion_r151838944 I don't think the benchmark really provides too much value, and was overlapping in responsibility with was was necessary so I simplified it.